src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -6544,11 +6592,16 @@ def start_sub_agents(self):
                         'task_slots'   : ['%s:0' % node], 
                         'task_offsets' : [], 
                         'lm_info'      : self._cfg['lrms_info']['lm_info']}
-                cmd, hop = agent_lm.construct_command(task_exec="/bin/sh",
-                        task_args="%s/bootstrap_2.sh %s" % (os.getcwd(), sa),
-                        task_numcores=1, 
-                        launch_script_hop='/usr/bin/env RP_SPAWNER_HOP=TRUE "%s"' % ls_name,
-                        opaque_slots=opaque_slots)
+                agent_cu = {
```

Comment:
More "kind of" than "is" :)
But thanks for the change!
Given that its now also used for the agents, it should ideally become CU/agent neutral.